### ISSUE TITLE
LEF file errors

### ISSUE THREAD

**Issue by mguthaus** [asyncvlsi/AMC#9]
There are some errors in the LEF file that were fixed in OpenRAM, but have not been ported to AMC. 

1) For example, library cells are entirely blocked rather that detailed blockages which will cause pin access errors in P&R.

2) In addition, the LEF file units are off by a few orders of magnitude which will result in a *meter* long memory sizes.

**Comment by sataei** [comment]
Yes. We are aware of this issue. Right now, we are reconstructing the entire compiler and will address this issue in next release.

**Comment by sataei** [comment]
Thank you. We are aware of this issue. Right now, we are  restructuring the entire compiler and will address this issue in next release.
